WeatherMonitor/Bots/Bot.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using WeatherMonitor.ConfigData;
+using WeatherMonitor.PrintConfig;
+using WeatherMonitor.WeatherManagement;
+namespace WeatherMonitor.Bots;
+
+public abstract class Bot
+{
+    [JsonProperty("enabled")]
```

Comment:
because the Json configuration file has the properties in the lowercase but the class props name have to  start with upper case. so the mapping between them did not work till I match the names using this Attribute.